src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -583,6 +625,11 @@ ConvSolution InitInvokerFactoryNCHW(const ExecutionContext& ctx,
             handle.ResetKernelTime();
 
             const auto& data_ctx = primitive_parameters.CastTo<CastType>();
+            if constexpr(std::is_same<CastType, miopen::conv::WrWInvokeParams>::value)
+            {
+                auto zero = 0.0f;
+                SetTensor(handle, data_ctx.tensors.dwDesc, data_ctx.tensors.dw, &zero);
```

Comment:
isn't this will clear zero of all the wrw solvers? which will result in perf regression